src/config.c

```diff
@@ -36,8 +38,6 @@
 #include "input/mpegts/scanfile.h"
 
 #include <netinet/ip.h>
-#define COMPAT_IPTOS
-#include "compat.h"
```

Comment:
To be precise. The error above appears with your fix if the lines are not moved. By moving the lines as suggested in my commit the error goes away.